### DISCUSSION TITLE
Class request: vehicle identification number

### DISCUSSION THREAD

**Discussion by jonathanvajda** [CommonCoreOntology/CommonCoreOntologies#389] [Ontology Terms and Class Schema]
There is a wide range of use cases where vehicles are described with respect to their vehicle identification number (VIN), and this assists with entity resolution. This seems useful for manufacturing, commerce, government, university, and other contexts. I think this makes a good case for it being included in the common core rather than a lower-level ontology.

VIN’s follow a standard, and there are competing standards. Here’s from Wikipedia:

> There are at least four competing standards used to calculate the VIN.
> 
> - [FMVSS](https://en.wikipedia.org/wiki/FMVSS) 115, Part 565: Used in United States and Canada
> - [ISO](https://en.wikipedia.org/wiki/International_Organization_for_Standardization) 3779: Used in Europe and many other parts of the world
> - [SAE](https://en.wikipedia.org/wiki/SAE_International) J853: Very similar to the ISO standard
> - [ADR](https://en.wikipedia.org/wiki/Australian_Design_Rules) 61/2 used in Australia, referring to ISO 3779 and 3780

For government, here’s the US https://www.ecfr.gov/current/title-49/subtitle-B/chapter-V/part-565

I think at minimum, “Vehicle Identification Number” should be in CCO, and whether to include the classification standards in the model that may require more debate.

I am not opposed to including “World Manufacturer Identifier” as well, since every VIN has a WMI as a proper part.

Recommended assertions:
```
rdfs:subClassOf cco:CodeIdentifier ;
rdfs:label “Vehicle Identification Number”@en ;
cco:alternative_label “vehicle chassis number”@en ,
	“vehicle frame number”@en ;
cco:acronym “VIN”@en ;
cco:curated_in_ontology Artifact Ontology
```

Caveat:
From what I see in CCO’s modeling of UPC, EAN, ISBN, and ISSN, the recommendation may be that what we want a “Vehicle identification Bar Code”, which is ultimately under the class “Information bearing artifact.”

I’m willing to go that route. But I’d want to see some documentation recommending the design pattern.
```
rdfs:subClassOf cco:OneDimensionalBarcode ;
rdfs:label “Vehicle Identification barcode”@en ;
cco:alternative_label “vehicle chassis barcode”@en ,
	“vehicle frame barcode”@en ;
cco:curated_in_ontology Artifact Ontology
```

**Comment by jimschoening1** [comment]
In my work on P3195.1.2 Person Ontology, I created an Orphan Ontology of classes a person would want to store in their Personal Data Store (PDS), but don't below in the Person Ontology.  It includes VIN and these other classes (which Mark Jensen drafted for prior to my retirement).  So I'd like them to find a home in CCO or some new domain.  

VehicleIdentificationNumber

https://opensource.ieee.org/person-ontology-group/person-ontology-project/-/blob/master/dev/Orphan-Ontology-Dev.ttl?ref_type=heads#L449  

AutomobileModel

https://opensource.ieee.org/person-ontology-group/person-ontology-project/-/blob/master/dev/Orphan-Ontology-Dev.ttl?ref_type=heads#L112

AutomobileTrimLevel

https://opensource.ieee.org/person-ontology-group/person-ontology-project/-/blob/master/dev/Orphan-Ontology-Dev.ttl?ref_type=heads#L119

AutomotiveModelGeneration

https://opensource.ieee.org/person-ontology-group/person-ontology-project/-/blob/master/dev/Orphan-Ontology-Dev.ttl?ref_type=heads#L126

**Comment by neilotte** [comment]
@jonathanvajda @jimschoening1 As the discussions regarding ICEs have implications for this thread, I'm moving them into discussion. However, if there are targeted full term requests (including definitions, source, parent class), consider re-opening them as issues that we can target for a release.
